sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1531,6 +1537,37 @@ public void addContextTag(final @NotNull String contextTag) {
     this.contextTags.add(contextTag);
   }
 
+  /**
+   * Returns whether sending of client reports has been enabled.
+   *
+   * @return true if enabled; false if disabled
+   */
+  public @NotNull Boolean isSendClientReports() {
+    return sendClientReports;
+  }
+
+  /**
+   * Enables / disables sending of client reports.
+   *
+   * @param sendClientReports true enables client reports; false disables them
+   */
+  public void setSendClientReports(@NotNull Boolean sendClientReports) {
```

Comment:
Similar to https://github.com/getsentry/sentry-java/pull/1982/files#r845784220
And we can use primitive types

Reply:
Done